packages/core/src/Modules/Cashier/Containers/payment-agent.jsx

```diff
@@ -5,21 +5,19 @@ import PaymentAgentList from '../Components/payment-agent-list.jsx';
 import Virtual from '../Components/Error/virtual.jsx';
 import PaymentAgentWithdrawForm from '../Components/Form/payment-agent-withdraw-form.jsx';
 
-class PaymentAgent extends React.Component {
-    componentDidMount() {
-        this.props.setActiveTab(this.props.container);
-    }
+const PaymentAgent = ({ setActiveTab, container, is_virtual, verification_code, is_payment_agent_withdraw }) => {
+    React.useEffect(() => {
+        setActiveTab(container);
+    }, []);
```

Comment:
Did you check warnings on this effect while running `npm run test`? Don't we need some `eslint ignore` commands here?